dino.doap.in

```diff
@@ -202,5 +273,26 @@
         <xmpp:status>complete</xmpp:status>
       </xmpp:SupportedXep>
     </implements>
+    <implements>
+      <xmpp:SupportedXep>
+        <xmpp:xep rdf:resource="https://xmpp.org/extensions/xep-0391.html" />
+      </xmpp:SupportedXep>
+    </implements>
+    <implements>
+      <xmpp:SupportedXep>
+        <xmpp:xep rdf:resource="https://xmpp.org/extensions/xep-0398.html" />
+        <xmpp:status>complete</xmpp:status>
+      </xmpp:SupportedXep>
+    </implements>
+    <implements>
+      <xmpp:SupportedXep>
+        <xmpp:xep rdf:resource="https://xmpp.org/extensions/xep-0402.html" />
```

Comment:
Dino has code that does 402, but doesn't actually use it in practive yet. Same for 0223.

Reply:
Shall I put them in as "planned" or drop that commit?

Reply:
I'd prefer to just leave them out for now.

Reply:
Ok, dropping that commit.

Reply:
... apparently I dropped the wrong commit. git :cry:

Reply:
Now then?